quarkus/README.adoc

```diff
@@ -117,7 +117,6 @@ All consumers with the same group ID belong to the consumer group with that ID.
 $ export BOOTSTRAP_SERVER=__<bootstrap_server>__
 $ export CLIENT_ID=__<client_id>__
 $ export CLIENT_SECRET=__<client_secret>__
-$ export GROUP_ID=__<group_id>__
```

Comment:
If it doesn't work with Quarkus, are we going to remove the other doc too?
_and consumer group ID as_
_All consumers with the same group ID belong to the consumer group with that ID._

Reply:
not sure about kafkacat and bin scripts - for those maybe we want to say explicitly what group to use

Reply:
I mean the mentions of consumer groups in the same quarkus quickstart. Above the id you took out.^^